lib/symboldatabase.cpp

```diff
@@ -1425,6 +1428,49 @@ void SymbolDatabase::createSymbolDatabaseEscapeFunctions()
     }
 }
 
+void SymbolDatabase::createSymbolDatabaseExprIds()
+{
+    MathLib::bigint base = 0;
+    // Find highest varId
+    for (const Variable *var : mVariableList) {
+        if (!var)
+            continue;
+        base = std::max<MathLib::bigint>(base, var->declarationId());
+    }
+    for (const Scope * scope : functionScopes) {
+        MathLib::bigint id = base+1;
```

Comment:
Yea, this was intentional because I am concerned we might run out of id numbers, but maybe not since this integer is large enough to assign a unique id to every token. I think I will change this.